app/views/foods/search.html.erb

```diff
@@ -0,0 +1,19 @@
+<nav class="navbar navbar-default" role="navigation">
+  <div class="navbar-header">
+    <%= link_to "Ingredient Search", root_path, class: "navbar-brand" %>
+  </div>
+  <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+    <ul class="nav navbar-nav">
+      <%= form_tag :foods, method: :get, class: "form-inline" do %>
+        <div class="form-group search-field">
+          <%= text_field_tag(:q) %>
+          <%= submit_tag "Search", class: "btn btn-primary" %>
+        </div>
+      <% end %>
+    </ul>
+  </div>
+</nav>
+
+<div id="search-results-count">
+    <h2><%= @results %></h2>
+<div>
```

Comment:
You will need a new view for your foods page. The view that should be rendered after a search is performed should be the view for the Foods#Search action.